.github/workflows/espresso-devnet-tests.yaml

```diff
@@ -60,28 +77,14 @@ jobs:
           docker compose build
           docker compose pull l1-validator espresso-dev-node l1-data-init
 
-      - name: Run Smoke test without TEE
-        run: go test -timeout 30m -p 1 -count 1 -run 'TestSmokeWithoutTEE' -v  ./espresso/devnet-tests/...
-
-      - name: Run Challenge Game test
-        run: go test -timeout 30m -p 1 -count 1 -run 'TestChallengeGame' -v  ./espresso/devnet-tests/...
-
-      - name: Run Withdraw test
-        run: go test -timeout 30m -p 1 -count 1 -run 'TestWithdrawal' -v  ./espresso/devnet-tests/...
-
-      - name: Run Batcher Restart test
-        run: go test -timeout 30m -p 1 -count 1 -run 'TestBatcherRestart' -v  ./espresso/devnet-tests/...
-
-      - name: Run Change Batch Inbox Owner test
-        run: go test -timeout 30m -p 1 -count 1 -run 'TestChangeBatchInboxOwner' -v  ./espresso/devnet-tests/...
-
       - name: Build Devnet with TEE
+        if: matrix.tee
         run: |
           cd espresso
           COMPOSE_PROFILES=tee docker compose build
 
-      - name: Run Smoke test with TEE
```

Comment:
Side note: I see that the CI is running, but @miguelCyclone mentioned the smoke test CI failed on his PR, so maybe it'll also fail here.